internal/services/zero_trust_organization/resource.go

```diff
@@ -194,13 +204,62 @@ func (r *ZeroTrustOrganizationResource) Read(ctx context.Context, req resource.R
 	}
 	data = &env.Result
 
+	var stateData *ZeroTrustOrganizationModel
+	resp.Diagnostics.Append(req.State.Get(ctx, &stateData)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	resp.Diagnostics.Append(normalizeReadZeroTrustOrganizationAPIData(ctx, data, stateData)...)
 	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
 }
 
 func (r *ZeroTrustOrganizationResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
 
 }
 
+func (r *ZeroTrustOrganizationResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
```

Comment:
This should be fine, but just wanted to call out that imports _should_ be automatic. If they don't exist automatically, it's usually an indication of a mis-alignment of path parameters and content in the response body.

Reply:
The /organizations API is anything but restful

like to get a single API, you call /organizations, which only returns a single item and not a list

Anyway, we should fix in TF as we're not gonna be able to change the API now